sycl/include/CL/sycl/detail/defines_elementary.hpp

```diff
@@ -100,3 +100,20 @@
 // clang emits "warning:" in the message pragma output
 #define __SYCL_WARNING(msg) __pragma(message(msg))
 #endif // __GNUC__
+
+// defined __SYCL_UNROLL to add pragma/attribute unroll to a loop
+#ifndef __SYCL_UNROLL
+#if defined(__clang__)
+#define STRINGIFY(x) #x
```

Comment:
This should not be redefined three different times. Also, if this is the approach you want to take, there's some other uses of the stringize operator that could be replaced in this file (rather than using `__SYCL_GCC_PRAGMA` in some places and this approach in others).

Reply:
Thanks! Applied (guess tho __SYCL_GCC_PRAGMA is the only definition to change in the file)